recipes/wimpy/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "wimpy" %}
+{% set gh_author = "wimglenn" %}
+{% set version = "0.6" %}
+{% set sha256 = "014d21a31980eab48b92c4210e9fdba643b79f7081c64ea9bc97f2be9db9bd58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ gh_author }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Avoiding variables which are only used once makes it easier for the bot to make updates:

```suggestion
  url: https://github.com/wimglenn/{{ name }}/archive/v{{ version }}.tar.gz
  sha256: 014d21a31980eab48b92c4210e9fdba643b79f7081c64ea9bc97f2be9db9bd58
```

(it should handle this scenario regardless but as I had other comments I might as well include this one)